subsquid/src/vestingProcessor.ts

```diff
@@ -59,15 +59,17 @@ export async function processVestingScheduleAddedEvent(
   ctx: EventHandlerContext,
   event: VestingVestingScheduleAddedEvent
 ) {
-  const { to, asset, schedule } = getVestingScheduleAddedEvent(event);
+  const { from, to, asset, schedule } = getVestingScheduleAddedEvent(event);
 
-  const beneficiary = encodeAccount(to);
+  const toAccount = encodeAccount(to);
+  const fromAccount = encodeAccount(from);
 
   const vestingSchedule = new VestingSchedule({
     id: randomUUID(),
+    from: fromAccount,
     eventId: ctx.event.id,
-    scheduleId: `${beneficiary}-${asset.toString()}`,
-    beneficiary,
+    scheduleId: `${toAccount}-${asset.toString()}`,
+    to: toAccount,
```

Comment:
is this the same as beneficiary? `to` is a bit ambiguous, unless we add a comment line explaining.

Reply:
`scheduleId` is not the same as beneficiary, it is a value generated here to create unique ids for vesting schedules as they did not have them previously on the pallet